modules/extension/mbstyle/src/main/java/org/geotools/mbstyle/expression/MBLookup.java

```diff
@@ -56,22 +56,26 @@ public Expression lookupAt() {
     public Expression lookupGet() {
         if (json.size() == 2 || json.size() == 3) {
             if (json.size() == 2) {
+                // Example: ["get", "propertyName"]
                 if (parse.isString(json, 1)) {
                     String propertyName = parse.get(json, 1);
                     return ff.property(propertyName);
-                } else {
+                }
+                // Example: ["get", key]
+                else {
```

Comment:
Pretty sure splitting the closing brace from the `else` violates the style guide (although Travis CI seems happy with it, so I guess not?). Not really important either way.